scripts/lighthouse/format-results.js

```diff
@@ -0,0 +1,50 @@
+function formatResults({ desktop, mobile }) {
+    const desktopResult = formatResult(desktop, '🖥️ Desktop');
+    const mobileResult = formatResult(mobile, '📱 Mobile');
+
+    return `### ⚡️ Lighthouse results
+
+${desktopResult}
+
+${mobileResult}
+    `;
+}
+
+function formatResult({ manifest, links }, title) {
```

Comment:
Should this be declared above line 2?

Reply:
These are function declarations so ordering doesn't matter as the functions are hoisted.